### ISSUE TITLE
Data preparation

### ISSUE THREAD

**Issue by yuchen2580** [VisionLearningGroup/DA_Detection#10]
Hi, 

Im trying to use this repo to reproduce everything you did in your paper,
However, the watermark & clipart data you use is from another repo 'cross-domain-detection'

The other repo need Cupy to generate data (watermark and clipart). For some reasons (working environment) I cannot install Cupy correctly. (conflict on cudatoolkit and the cupy). Would it be possible to share your training data on dropbox?

Also, the watermark data 's annotations are generated using ssd300 in other repo, is it true?

**Comment by yuchen2580** [comment]
Ok, so only test set are fully annotated while train set for watercolor only have dummy annotations. Closing this issue.
